src/schema/paddle.ts

```diff
@@ -118,6 +130,121 @@ export const typeDefs = /* GraphQL */ `
   extend type Query {
     pricePreviews: PricePreviews! @auth
     corePricePreviews: PricePreviews! @auth
+    plusPricingMetadata(variant: String): [PlusPricingMetadata!]! @auth
```

Comment:
Please make sure old endpoints works as is after deployment.

Reply:
Yeah, we didn't touch anything existing so we can easily deprecate it when the time comes.